### DISCUSSION TITLE
[Feature addition] Integrating ChatGPT into FreeCAD

### DISCUSSION THREAD

**Discussion by MrMathematician** [FreeCAD/FreeCAD#10615] [Code contribution discussions]
I was considering trying to integrate the ChatGPT API into FreeCAD while also adding specific features that would help in the design process of parts. This includes recommending certain dimensions based on the function and required specifications for the part, recommending a certain order of work to make the modelling process easier or recommending changes in certain parts of the design for enhancing performance. I was wondering if this feature is useful or is effective enough for users so that I can start working on it.

**Comment by luzpaz** [comment]
Check the forum..there are a few ongoing threads and efforts discussing this.

**Reply by MrMathematician** [reply]
I will. Thanks for the feedback.
